tests/functional/aws-node-sdk/test/multipleBackend/delete.js

```diff
@@ -64,7 +71,18 @@ describe('Multiple backend delete', () => {
                     `Expected success, got error ${JSON.stringify(err)}`);
                 s3.getObject({ Bucket: bucket, Key: fileObject }, err => {
                     assert.strictEqual(err.code, 'NoSuchKey', 'Expected ' +
-                        'error but got sucess');
+                        'error but got success');
+                    done();
+                });
+            });
+        });
+        it('should delete object from AWS', done => {
+            s3.deleteObject({ Bucket: bucket, Key: awsObject }, err => {
```

Comment:
please add a test where you have an object foo on the aws backend and then you put foo on the mem backend.  The foo on aws should be deleted.  Also add a test the other way around -- put foo on mem and then foo on aws.